python/cudnn/gemm/frost/kernel_templates/sm103_block_scale_matmul_2ctamma.py

```
@@ -1100,7 +1125,7 @@ def _kernel(
         # @@TMA_STORE_ONLY:END@@
 
         while is_valid != 0:
-            coord_m = tile_m * cgrp_tile_mnk[0] + m_rank * cta_tile_mnk[0]
+            coord_m_tile = tile_m * cgrp_tile_mnk[0] + m_rank * cta_tile_mnk[0]
             coord_n_c = tile_n * cgrp_tile_mnk[1] + n_rank * pair_n_size
             if cutlass.const_expr(cta_tile_mnk[0] == 64):
```

Ruff (0.16.1)

[error] Undefined name `cta_tile_mnk`

(F821)